fab/environments.yml

```diff
@@ -378,6 +377,163 @@ icds:
       XFormToElasticsearchPillow:
         num_processes: 4
 
+icds-new:
+  environment: icds
```

Comment:
Should we name it different (e.g. `icds2`) so it doesn't conflict with `icds` if we deploy from local machine?

Reply:
yea, this should be `icds-new`